assets/dev-env.lando.template.yml.ejs

```diff
@@ -154,6 +156,10 @@ services:
           target: /scripts
           volume:
             nocopy: true
+#      environment:
+#        LANDO_NO_SCRIPTS: 1
+#        LANDO_NEEDS_EXEC: 1
+#    initOnly: true
```

Comment:
The commented-out lines will need to be uncommented in 2.29.

Reply:
Is this still the case. Could you add more context to this please?

Reply:
I failed to sort out the issue with permissions, will leave this for another PR.